onnxruntime/core/optimizer/attention_fusion.cc

```diff
@@ -281,7 +280,7 @@ Status AttentionFusion::ApplyImpl(Graph& graph, bool& modified, int graph_level,
     ORT_RETURN_IF_ERROR(Recurse(node, modified, graph_level, logger));
 
     if (node.GetOutputEdgesCount() == 4 &&
-        graph_utils::IsSupportedOptypeVersionAndDomain(node, "LayerNormalization", {1}, kOnnxDomain) &&
+        graph_utils::IsSupportedOptypeVersionAndDomain(node, "LayerNormalization", {9}, kOnnxDomain) &&
```

Comment:
Scott McKay
This doesn't quite match with the change to the operator schema. Either you need to have 2 versions of the op (1 and 9), or any reference to version 1 here should go away as well.

Sherlock Huang
Removed 1.